<?xml version="1.0" encoding="UTF-8"?><Posts><Post><PostId>2390766</PostId><Url>http://stackoverflow.com/questions/2390766</Url><Stack><Type>UserWarning</Type><Message>Boo!</Message><Frame><File>stdin</File><Line>1</Line><Method>module</Method></Frame><Frame><File>stdin</File><Line>2</Line><Method>f</Method></Frame><Frame><File>stdin</File><Line>1</Line><Method>module</Method></Frame><Frame><File>stdin</File><Line>2</Line><Method>f</Method></Frame></Stack></Post><Post><PostId>2390766</PostId><Url>http://stackoverflow.com/questions/2390766</Url><Stack><Type>UserWarning</Type><Message>Boo!</Message><Frame><File>stdin</File><Line>1</Line><Method>module</Method></Frame><Frame><File>stdin</File><Line>2</Line><Method>f</Method></Frame><Frame><File>stdin</File><Line>1</Line><Method>module</Method></Frame><Frame><File>stdin</File><Line>2</Line><Method>f</Method></Frame></Stack></Post><Post><PostId>11363664</PostId><Url>http://stackoverflow.com/questions/11363664</Url><Stack><Type>UserWarning</Type><Message>uhoh</Message><Frame><File>test_warnings.py</File><Line>15</Line><Method>module</Method></Frame><Frame><File>test_warnings.py</File><Line>12</Line><Method>main</Method></Frame><Frame><File>test_warnings.py</File><Line>8</Line><Method>bar</Method></Frame><Frame><File>test_warnings.py</File><Line>5</Line><Method>foo</Method></Frame></Stack></Post><Post><PostId>16237927</PostId><Url>http://stackoverflow.com/questions/16237927</Url><Stack><Type>UserWarning</Type><Message>uh oh</Message><Frame><File>warning_to_exception.py</File><Line>23</Line><Method>module</Method></Frame><Frame><File>warning_to_exception.py</File><Line>9</Line><Method>func2</Method></Frame><Frame><File>warning_to_exception.py</File><Line>13</Line><Method>func2b</Method></Frame></Stack></Post></Posts>